sonar-project.properties

```diff
@@ -4,7 +4,7 @@ sonar.organization=govuk-one-login
 
 # Path is relative to the sonar-project.properties file. Replace "\" by "/" on Windows.
 sonar.sources=Sources
-sonar.tests=Tests,UITests
```

Comment:
are `UITests` not tests anymore? 👀

Reply:
Yeah I wasn't sure how sonar handles this, whether we just needed to point to the top folder in the directory of do a `Tests/*`... in which case I imagine Sources would have to be the same  i.e. `Sources/*`?

Reply:
Sounds like it should be okay if all the test files are within that folder, in a subfolder structure. that Tests folder now contains UnitTests, ConfigTests and UITests

![image 2023-11-07 at 10 14 29](https://github.com/alphagov/di-mobile-ios-onelogin-app/assets/90717175/5e001156-e896-4b10-bb0d-85ad3e7f8634)

Reply:
A second look at the documentation looks like it should be
`sonar.sources=Sources/`
`sonar.test=Tests/`
I've corrected the file now 👍